### PR TITLE
Radio Button: new DS6 styles

### PR DESCRIPTION
**Pull request by PrashantAshok**
## Description
- Apply DS6 styles & use new icons for radio buttons. 
- Create field folder for DS6 containing few styling rule changes to accommodate newer default icon sizes

## References
Issue #141 
screenshot-2018-5-1 ebay skin for ds6

### CODE REVIEW

**Comment by seangates** [comment]
@PrashantAshok So, make sure you use as much of the PR template as possible to explain what you're doing. Specifically, we need some details as to any implementation changes from the previous version, any improvements, and the new design as it stands now (e.g. screenshots).

Can you add a little more meat to the PR description?

File: docs/_includes/ds6/radio.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
+<div id="radio">
+    <h2><span class="secondary-text">@ebay/skin/</span>radio</h2>
+    <p>A radio button is a form control thats allow a user to select a single option from a group of choices.</p>
+    <p>The purpose of a radio button is to collect form data; therefore radios should always be used in conjunction with a form, label and submit button.</p>
```
Comment (seangates) [inline docs/_includes/ds6/radio.html]: Minor improvement:

>The purpose of a radio button is to collect form data. Therefore, radio buttons should always be used in conjunction with a form, label, and submit button.

File: docs/_includes/ds6/radio.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+    <h3>Default Radio</h3>
+    <p>Use the <span class="highlight">radio</span> base class to create a radio.</p>
```
Comment (seangates) [inline docs/_includes/ds6/radio.html]: For clarity:
>Use the <span class="highlight">radio</span> base class to create a styled radio.

Comment (seangates) [reply docs/_includes/ds6/radio.html]: Oh, and make sure to keep the `highlight` span.

File: docs/_includes/ds6/radio.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                    <svg aria-hidden="true" class="radio__checked" focusable="false">
+                        <use xlink:href="#icon-confirmation"></use>
+                    </svg>
+                </span>
+            </span>
+        </div>
+    </div>
+
+    {% highlight html %}
+<span class="field__control radio" style="color: #5ba71b">
```
Comment (seangates) [inline docs/_includes/ds6/radio.html]: Use a DS6 available color. For example: #71c63e (which is also green).

Comment (seangates) [reply docs/_includes/ds6/radio.html]: Can you take care of this one (and the corresponding one on line 145)? ☝️

**Comment by ianmcburnie** [comment]
@PrashantAshok You have 1 conflicting file. You may need to run `yarn build` again, otherwise maybe a rebase or merge of 4.1.0

**Comment by ianmcburnie** [comment]
Not sure why, but it looks like there is a *little* too much distance between the icon and label

In DS4:

screen shot 2018-05-04 at 10 38 41 am

In this PR:

screen shot 2018-05-04 at 10 38 49 am

@PrashantAshok Could you please investigate.

**Comment by ianmcburnie** [comment]
The issues above will probably apply to your checkbox branch too, so I recommend that you fix this branch first. When we merge it, we can rebase checkbox branch with the 4.1.0 branch.

File: dist/field/ds6/field.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
+span.field {
```
Comment (ianmcburnie) [inline dist/field/ds6/field.css]: @PrashantAshok You'll need to remove this dist file now.

File: dist/radio/ds6/radio.css [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+.radio__control[type="radio"],
+.radio__custom-control[type="radio"] {
+  font-size: 100%;
+  height: 1.125em;
+  opacity: 0;
+  position: absolute;
+  width: 1.125em;
+  z-index: 1;
+}
+.radio__custom-control[type="radio"]:not([disabled]) {
```
Comment (ianmcburnie) [inline dist/radio/ds6/radio.css]: We could maybe make better use of grouped selectors for the next 20 lines or so, but I don't thin it's a blocker. @seangates thoughts?

**Comment by ianmcburnie** [comment]
@PrashantAshok I have been looking at this some more, since I discovered we have replaced and simplfied the "merged" inline SVG case (i.e a single symbol containing both the checked and unchecked paths) for two separate SVGs, and I *think* we can now do away with the `radio__custom-control` altogether (since that class was created for the merged paths use case). I will do a bit more tinkering and try and send out a PR soon. Any changes would need to be also reflected in the checkbox PR.

**Pull request merged**